Packs/FeedMitreAttackv2/Playbooks/Mitre_Attack_-_Extract_Technique_Information_From_ID.yml

```diff
@@ -199,8 +211,50 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 290,
-          "y": 30
+          "x": 120,
+          "y": -190
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "8":
+    id: "8"
+    taskid: e013809e-88cf-4415-87cf-32fe291b92a7
+    type: condition
+    task:
+      id: e013809e-88cf-4415-87cf-32fe291b92a7
+      version: -1
+      name: Is there a Mitre Attack Technique ID?
```

Comment:
Found any MITRE Attack Technique ID's?

Reply:
May I suggest `Is Mitre Attack Technique ID provided?` instead?
The reason is that this condition checks against the inputs and not command results or so.

Reply:
Of course :), but it can be multiple MITRE ID's right? And please write MITRE and not Mitre.

Reply:
Yes, it can handle multiple values, I can make it plural (even though, tech writers usually ask to keep it single).
Other tasks are already set to plural so I think your suggestion is better in this case:)

Reply:
Done.